graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/DataLoaderProvider.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs
+
+import org.dataloader.DataLoaderRegistry
+import java.util.function.Supplier
+
+interface DataLoaderProvider {
```

Comment:
Could you please rename this to `DgsDataLoaderProvider`, this will make it consistent with other components in the  package.